projects/control-service/projects/pipelines_control_service/src/integration-test/README.md

```diff
@@ -3,13 +3,26 @@ This directory contains Integration tests for the Pipelines Control Service.
 The goal of the tests is to validate that the functionality of the service
 works in combination with other dependant components (KDC, Kubernetes, DB, Git repo, etc).
 
-# Prerequisites
+# Prerequisites to run locally
 * Kubernetes 1.15+ (E.g. minikube, kubernetes-in-docker, etc.)
 * Valid kubeconfig.yaml in ```${HOME}/.kube/config``` or set the ```datajobs.deployment.k8s.kubeconfig``` and ```datajobs.control.k8s.kubeconfig``` property
   in [integration-test/resources/application-test.properties](./resources/application-test.properties)
+* Git repo for storing the jobs
```

Comment:
git server is an actual prerequisite;
and git repository provisioned;

Reply:
changed